### PR TITLE
Increase data transfer progress interval to 10 s

### PR DESCRIPTION
**Pull request by 0x2b3bfa0**
See https://github.com/iterative/terraform-provider-iterative/pull/467#discussion_r839871754

### CODE REVIEW

File: task/common/machine/storage.go [hunk 1]
```diff
@@ -132,7 +132,7 @@ func Transfer(ctx context.Context, source, destination string, include string) e
 		return err
 	}
 
-	defer progress(2 * time.Second)()
+	defer progress(10 * time.Second)()
```
Comment (DavidGOrtega) [inline task/common/machine/storage.go:135]: Why 10? The transfer log might be also synced here? Is there any chance that the logs might be repeated?

Comment (0x2b3bfa0) [reply task/common/machine/storage.go:135]: It's just to reduce noise

Comment (0x2b3bfa0) [reply task/common/machine/storage.go:135]: Instead of updating the progress every 2 seconds, it updates the progress every 10 seconds.

Comment (0x2b3bfa0) [reply task/common/machine/storage.go:135]: See https://github.com/iterative/terraform-provider-iterative/pull/467#discussion_r839871754 for background.

Comment (casperdcl) [reply task/common/machine/storage.go:135]: - can we make it user-configurable somehow?
- can we merge it with the "still creating/destroying" messages?

Comment (0x2b3bfa0) [reply task/common/machine/storage.go:135]: > _can we merge it with the "still creating/destroying" messages?_

Yes, forking Terraform Core or with `LD_PRELOAD=do_this.so` 🙃

Comment (0x2b3bfa0) [reply task/common/machine/storage.go:135]: > _can we make it user-configurable somehow?_

Yes, either as part of the schema or through environment variables, as per https://github.com/iterative/terraform-provider-iterative/pull/467#discussion_r839961885. I wonder if this kind of parameters are important enough to make them configurable.

**Review by casperdcl — APPROVED** [review]

File: task/common/machine/storage.go (hunk 1 above)
Comment (0x2b3bfa0) [reply task/common/machine/storage.go:135]: ✅ Merged

**Pull request merged**